tools/api-generator/fixtures/06-spec-with-cli-specific-extensions.yaml

```diff
@@ -338,6 +344,10 @@ components:
       description: Flag that indicates whether the response body should be in the <a href="https://en.wikipedia.org/wiki/Prettyprint" target="_blank" rel="noopener noreferrer">prettyprint</a> format.
       in: query
       name: pretty
```

Comment:
does pretty have any meaning for the CLI?

Reply:
This is a fixture in a unit test, I don't think the actual values are relevant here.

Reply:
as in isn't the CLI always pretty?

Reply:
oh interesting seems like no after trying thi s

Reply:
But to reply to your question, it does not have a meaning to the CLI; it's just a regular parameter passed to the backend, and the CLI doesn't handle it any differently.